views/views_image.templ

```diff
@@ -32,7 +33,7 @@ templ ImageFitContain(ImageData, imageFit string) {
 	/>
 }
 
-func ImageLocation(info immich.ExifInfo) string {
+func ImageLocation(viewData ViewData, info immich.ExifInfo) string {
```

Comment:
I would probably change this to accept the slide of countries to hide, rather than the whole viewData.